### PR TITLE
#166187806 Empty react project setup

### PR DESCRIPTION
**Pull request by Emmanuelaaron**
#### What does this PR do?
Sets up an empty react project

#### Description of task completed
* Initialized the project
* Installed webpack
* Installed webpack-cli
* Installed babel plus it's required dependencies
* Installed react and react-dom
* Installed webpack-dev-server

#### How should this be manually tested on your local machine
* Do a `git fetch origin ch-empty-react-project-166187806`
* Do a `git checkout ch-empty-react-project-166187806`
* Install all the dependencies using `npm install`
* Then run the project using `npm start`
finishes [166187806](https://www.pivotaltracker.com/n/projects/2350340/stories/166187806)

[Finishes #166187806]

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" >
```
Comment (HabibSentongo) [inline dist/index.html]: @Emmanuelaaron why do we have this bootstrap link here yet we are to use SASS

**Review by HabibSentongo — CHANGES_REQUESTED** [review]

File: dist/index.html (hunk 1 above)
Comment (Emmanuelaaron) [reply dist/index.html]: This was just an oversight. let me remove it.

**Review by neelxie — APPROVED** [review]
As shown below I have tested the application and everything seems good.

Screenshot 2019-07-01 at 17 03 00

Screenshot 2019-07-01 at 17 03 18

File: dist/index.html (hunk 1 above)
Comment (Emmanuelaaron) [reply dist/index.html]: @HabibSentongo , I have realized that that link was created automatically only that I needed to include the dist folder inside the .gitignore file

Comment (HabibSentongo) [reply dist/index.html]: Alright great, other things are ok according to me

**Review by HabibSentongo — APPROVED** [review]

**Review by alexxsanya — APPROVED** [review]
@Emmanuelaaron good job, After following through the procedures you provided I have been able to set up the react app and it's running well.

**Review by jennizalwango — COMMENTED** [review]
l have been reviewing your and everything is good,
Thanks for the good work.

**Review by jennizalwango — COMMENTED** [review]
I am just wondering l have not seen eslint configured in any of the files. Please correct me if l am wrong aren't we supposed to config that as well during set up?

**Comment by Emmanuelaaron** [comment]
https://github.com/andela/ah-frontend-realers/pull/1#pullrequestreview-256497486

Thanks @jennizalwango  for taking note of that. I will be including it as I setup redux

**Comment by Emmanuelaaron** [comment]
> I am just wondering l have not seen eslint configured in any of the files. Please correct me if l am wrong aren't we supposed to config that as well during set up?

@jennizalwango  I have resolved this!!

**Comment by jennizalwango** [comment]
> > I am just wondering l have not seen eslint configured in any of the files. Please correct me if l am wrong aren't we supposed to config that as well during set up?
> 
> @jennizalwango I have resolved this!!

ohh ok @Emmanuelaaron  let me check it out

**Review by jennizalwango — COMMENTED** [review]
l have tried the application and so far its working Good work @Emmanuelaaron

**Review by alexxsanya — CHANGES_REQUESTED** [review]
Screenshot 2019-07-02 at 12 39 02

@Emmanuelaaron I happen to get  2 linter issues/errors as you may see in the above snippet.
1 - Component should be written as a pure function.
2 - JSX not allowed in a file with an extension of `.js `

Kindly action on this.

**Comment by Emmanuelaaron** [comment]
> Screenshot 2019-07-02 at 12 39 02
> 
> @Emmanuelaaron I happen to get 2 linter issues/errors as you may see in the above snippet.
> 1 - Component should be written as a pure function.
> 2 - JSX not allowed in a file with an extension of `.js `
> 
> Kindly action on this.

Thanks @alexxsanya  for pointing out this.
I have rectified the first issue but the second issue is turning out to be a blocker.
I can't really figure out what the problem is irrespective of installing `@babel/preset-react` responsible for compiling `JSX and other stuff down to Javascript`

**Comment by Emmanuelaaron** [comment]
> > Screenshot 2019-07-02 at 12 39 02
> > @Emmanuelaaron I happen to get 2 linter issues/errors as you may see in the above snippet.
> > 1 - Component should be written as a pure function.
> > 2 - JSX not allowed in a file with an extension of `.js `
> > Kindly action on this.
> 
> Thanks @alexxsanya for pointing out this.
> I have rectified the first issue but the second issue is turning out to be a blocker.
> I can't really figure out what the problem is irrespective of installing `@babel/preset-react` responsible for compiling `JSX and other stuff down to Javascript`

It is resolved

**Review by neelxie — APPROVED** [review]
@Emmanuelaaron working seamlessly on my side. Good work.

**Review by jennizalwango — APPROVED** [review]
Since you have implemented the feedback everything is working, l think your good to go

**Comment by alexxsanya** [comment]
> > > Screenshot 2019-07-02 at 12 39 02
> > > @Emmanuelaaron I happen to get 2 linter issues/errors as you may see in the above snippet.
> > > 1 - Component should be written as a pure function.
> > > 2 - JSX not allowed in a file with an extension of `.js `
> > > Kindly action on this.
> > 
> > 
> > Thanks @alexxsanya for pointing out this.
> > I have rectified the first issue but the second issue is turning out to be a blocker.
> > I can't really figure out what the problem is irrespective of installing `@babel/preset-react` responsible for compiling `JSX and other stuff down to Javascript`
> 
> It is resolved
@Emmanuelaaron I have checked it out and its now fine.

**Review by alexxsanya — APPROVED** [review]

**Pull request merged**